docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -24,19 +24,13 @@ The following packages are optional, but recommended:
 
 You can run Zanata on JBoss EAP 6 or Wildfly. Just download one of the installer archives below for your platform, and then extract it on top of your JBoss or Wildfly installation.
 
-- [Zanata for JBoss EAP](http://sourceforge.net/projects/zanata/files/installer/zanata-3.6.0-eap-6.zip/download)
-- [Zanata for Wildfly](http://sourceforge.net/projects/zanata/files/installer/zanata-3.6.0-wildfly-8.1.zip/download)
+- [Zanata for JBoss EAP or Wildfly](https://github.com/zanata/zanata-server/releases)
```

Comment:
We need to make it clear that the war file (binary) is to be downloaded, not the zip (source files).